theano/tensor/elemwise.py

```diff
@@ -125,6 +125,7 @@ class DimShuffle(Op):
 
     _f16_ok = True
     check_input = False
+    __props__ = ("input_broadcastable", "new_order")
```

Comment:
The tests are not meant to be run with `device=gpu`, but will still use the GPU if one is available (as long as you do not set `force_device=True`).
If you want to select which GPU is going to be used if needed, you can run the tests with `init_gpu_device=gpu2` for instance.